lib/app.js

```diff
@@ -0,0 +1,20 @@
+const { createServer } = require('http');
+const { getFilePath } = require('./utils/filePath');
+const fs = require('fs');
+const { parse } = requite('url');
+
+
+const app = createrServer((req, res) => {
+  const { pathname } = parse(req.url);
+  fs.readFile(getFilePath(pathname), { encoding: 'utf8' }, (err, content) => {
+    if(err && err.code === 'ENOENT') {
+      console.log(err);
+      res.statusCode = 404;
+      res.end('NotFount');
```

Comment:
'Not Found' this is the text that shows up in your browser if a user comes to a path that doesn't exist.